source/agora/test/Base.d

```diff
@@ -2631,12 +2631,12 @@ public final class LocalRestDNSResolver : DNSResolver
 
         Params:
             msg = DNS message
+            tcp = Query the server with TCP, default is UDP
 
     ***************************************************************************/
 
-    public override ResourceRecord[] query (Message msg) @trusted
+    public override ResourceRecord[] query (Message msg, bool tcp = false) @trusted
```

Comment:
Looks like this belongs to the previous commit, because otherwise it's not an override.